backend/queries/admin.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../db/db');
+
+const userQueries = require('./users');
+
+const getUserByEmail = async (email) => {
+    return await db.one('SELECT * FROM administration WHERE a_email = $1', email)
+}
+
+const addAdmin = async (firstName, LastName, email, password) => {
```

Comment:
@AminesCodes is there a reason LastName is capitalized while firstName is not?

Reply:
Good catch, sorry it’s a typo it should be lastName